tests/end_to_end/test_docker_monitoring/data/test_cases/cases_docker_monitoring.yaml

```diff
@@ -0,0 +1,51 @@
+- name: docker_pull
+  description: Test if an alert is generated when using the `pull` command.
+  configuration_parameters: null
+  metadata:
+    rule.id: 87932
+    rule.description: 'Docker: Image or repository nginx pulled'
+    rule.level: 3
+    extra:
+      data.docker.Action: pull
+    extra_vars:
+      event_description: Run 'docker pull nginx'
+      command: docker pull nginx
+
+- name: docker_run
+  description: Test if an alert is generated when using the `run` command.
+  configuration_parameters: null
+  metadata:
+    rule.id: 87903
+    rule.description: 'Docker: Container nginx_container started'
+    rule.level: 3
+    extra:
+      data.docker.Action: start
+    extra_vars:
+      event_description: Run 'docker run -d -P --name nginx_container nginx'
+      command: docker run -d -P --name nginx_container nginx
+
+- name: docker_exec
+  description: Test if an alert is generated when using the `exec` command.
+  configuration_parameters: null
+  metadata:
+    rule.id: 87908
+    rule.description: 'Docker: Started shell session in container nginx_container'
+    rule.level: 5
+    extra:
+      data.docker.Action: 'exec_start: /bin/bash '
+    extra_vars:
+      event_description: Run 'docker exec -d nginx_container /bin/bash'
+      command: docker exec -d nginx_container /bin/bash
+
+- name: docker_stop_rm
+  description: Test if an alert is generated when using the `stop` and `rm` commands.
```

Comment:
In this case, it only proves that it is destroyed, no? According to what you say, it is proved that it is stopped and destroyed, but really it is only proved that it is destroyed, and the fact of stopping it is necessary to be able to destroy it.

I would indicate that only the case of destroying is tested.

Reply:
@jmv74211 Ok, good point, thank you!

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2948/commits/010c9f5af10700f0f4060b31a88bcd43b03ccc02